products.js

```diff
@@ -18,23 +18,24 @@ const {readFileSync, writeFile, readdirSync, statSync} = require('fs');
 const {dirname, join} = require('path');
 const leven = require('leven');
 const {dom, props, out, rule, ruleset, score, type} = require('fathom-web');
+const {Annealer} = require('fathom-web/optimizers');
 const {staticDom} = require('fathom-web/utils');
 const tuningRoutines = {
-                        // 'title' : tunedTitleFnodes,
-                        // 'price' : tunedPriceFnodes,
-                        'image' : tunedImageFnodes
+                        // 'title' : {'routine': tunedTitleFnodes, 'coeffs': []},
+                        // 'price' : {'routine': tunedPriceFnodes, 'coeffs': []},
+                        'image' : {'routine': tunedImageFnodes, 'coeffs': [2.5, 2.5, 3, 0.5]}
                         };
 
-function tunedImageFnodes(nodeToCssMap) {
+function tunedImageFnodes(nodeToCssMap, coeffImgSize = 2.5, coeffImgHasSrc = 2.5, coeffImgTitle = 3, coeffKeywords = 0.5) {
     let title = '';
 
     function imageSize(fnode) {
       const css = nodeToCssMap.get(fnode.element);
-      return (css.left - css.right) * (css.top - css.bottom);
+      return (css.left - css.right) * (css.top - css.bottom) * coeffImgSize;
```

Comment:
Fortunately you're multiplying 2 negatives together, which cancels out the error. :-)